packages/playground/src/utils/delete_deployment.ts

```diff
@@ -21,11 +21,20 @@ export async function deleteDeployment(grid: GridClient, options: DeleteDeployme
     }
   }
 
-  /* Delete gateway */
+  /* Start Delete gateway */
+
+  /* For fvm/vm */
+  if (isVm(options.projectName)) {
+    await deleteVmGateways(grid);
```

Comment:
Changes should be applied for all gateway naming in the changed files.

Reply:
We already handling other solution
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1279/files/0217c0145c3decf2d3743ea2f3eaa35a2f1e5e5a#diff-8a184bc478d27b259428cae59743ec4826372f23d6241ad87264806ce51d865dR31